internal/storage_tar_ball.go

```diff
@@ -15,7 +16,7 @@ import (
 type StorageTarBall struct {
 	backupName  string
 	partNumber  int
-	size        int64
+	size        *int64
```

Comment:
technically, this is not size of tarball. This is a size of all tarballs.
I'm not sure this is a good place to store size of all tarballs, but let's leave it aside, just rename it correctly.